### PR TITLE
5763 print qr code label

### LINKED ISSUE #5763: Print QR Code Purpose isn't clear
## What went wrong? 😲

In Cold Chain Equipment Detail View we have a `Print QR Code` button
image

However, it actually prints an asset label containing a QR code.
Suggested change it to call it `Print Asset Label`

## Expected behaviour 🤔

Button says `Print Asset Label`

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Open Coldchain -> Equipment -> Select an Asset
2. See Print QR Code button

## Your environment 🌱

- Open mSupply Version: 2.4.1
- Legacy mSupply Central Server Version:

- Platform:

- Database type:

### PR DESCRIPTION
**Pull request by zachariah-at-msupply**
Fixes #5763 

# 👩🏻‍💻 What does this PR do?

Changes the text of the 'print qr' button.

This is to make it easier for the user to understand what the button does: previously it said 'Print QR', but it actually prints an asset label containing a QR code.

## 💌 Any notes for the reviewer?

Hello, I hope your day is going well? :)

As well as the text, I also changed the name of the translation elements from 'button.print-qr' and 'message.print-qr' and so on to '.print-asset-label', so that it made sense in light of the text. I changed that for all locales.
Will the text for those translation elements be updated as well?

# 🧪 Testing

- [ ] Open Coldchain -> Equipment -> Select an Asset
- [ ] See Print QR Code button now says Print Asset Label

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [x] **These areas should be updated or checked**: The instructions for printing asset labels or QR codes for coldchain equipment.

### CODE REVIEW

**Review by jmbrunskill — APPROVED** [review]
Thanks @zachariah-at-msupply

Good idea to update the function names and labels too.

As for the other translations, it would be good to get them translated at some point however, I think doing it just in english is ok for now.

**Comment by lache-melvin** [comment]
Hi - to get the other languages updated, please just delete the `print-qr/print-asset-label` entry from the common.json files for the other languages?

Weblate only shows untranslated values (when comparing English common.json with other language), rather than "changed" ones - so if we need something updated, best to remove it so it is flagged for translation again!

**Comment by zachariah-at-msupply** [comment]
> Hi - to get the other languages updated, please just delete the `print-qr/print-asset-label` entry from the common.json files for the other languages
> 
> Weblate only shows untranslated values (when comparing English common.json with other language), rather than "changed" ones - so if we need something updated, best to remove it so it is flagged for translation again!

Smashing, I did that.
I'll wait for the review, then merge it 👍

File: client/packages/common/src/intl/locales/en/common.json [hunk 1]
```diff
@@ -327,7 +327,7 @@
   "error.plugin": "There was an error loading the plugin {{name}}",
   "error.plugin-unavailable": "Plugin unavailable",
   "error.prescription-not-found": "Prescription not found.",
-  "error.printing-qr": "Error printing QR code",
+  "error.printing-asset-label": "Error printing asset label code",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/common.json:330]: Oo - could you make quick follow up PR @zachariah-at-msupply, think `code` needed to be removed here 🙏

Comment (zachariah-at-msupply) [reply client/packages/common/src/intl/locales/en/common.json:330]: All done, the follow up PR is here...
https://github.com/msupply-foundation/open-msupply/pull/5853

**Pull request merged**